package.json

```diff
@@ -21,11 +21,13 @@
     "client-oauth2": "2.2.0",
     "event-emitter": "^0.3.4",
     "guid": "0.0.12",
-    "pcmjs": "^0.0.2"
+    "pcmjs": "^0.0.2",
+    "rewire": "^2.5.2"
   },
   "devDependencies": {
     "babel-plugin-transform-object-assign": "^6.8.0",
-    "babel-preset-es2015": "^6.13.2",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
Travis apparently searches for this preset and breaks if it is missing. Not sure why.